libminiooni/libminiooni_unit_test.go

```diff
@@ -26,12 +26,29 @@ func TestFileInputInvalidFile(t *testing.T) {
 	// test function panics with invalid file
 	defer func() { recover() }()
 
+	dir += "/test.txt"
 	opts := Options{InputFilePath: dir}
 	loadFileInputs(&opts)
 
 	t.Errorf("expected panic with invalid input file")
 }
 
+func TestFileInputDirectory(t *testing.T) {
```

Comment:
Properly adapted, this could become a test for `fsx.Open`. (To be clear, `fsx.Open` should return `*os.File, error` rather than panicking as it does here, since it's a convenience/library function.)